R/tags.R

```diff
@@ -435,7 +435,7 @@ tag_tree <- function(.x) {
 
 parse_tags_tbl <- function(x) {
   parsed_tags <- purrr::map_dfr(x, ~ {
-    out <- dplyr::tibble(
+    out <- tibble::tibble(
```

Comment:
We can later, but is it worth doing the hacky add the class on without also requiring `tibble`?

Reply:
I'm not sure, tibble is used heavily throughout, they're the main thing that the API wrappers return. Can make an issue to evaluate.